### ISSUE TITLE
Chrome now blocks port 10080.

### ISSUE THREAD

**Issue by lmpater** [gogs/gogs#6572]
Check this article:
https://www.bleepingcomputer.com/news/security/google-chrome-blocks-port-10080-to-stop-nat-slipstreaming-attacks/

It may make sense to update the documentation that now suggests to map the port 10080 of the host to the 3000 of the container. I just spent about 1hr debugging the issue and, when I was close to give up, I tried to open Gogs on Safari and it worked - so I was able to narrow down the problem to Chrome and found out the information on the port restriction.

Btw, it seems Firefox has been blocking this port for a while now.

**Comment by djdomi** [comment]
Update: all newer Browsers blocks

- ports 
 - 69, 
 - 137, 
 - 161, 
 - 554, 
 - 1719, 
 - 1720, 
 - 1723, 
 - 5060, 
 - 5061, 
 - 6566.

To use them without a Paint use either a other port or a reverse-proxy like nginx

**Comment by kylegoetz** [comment]
FWIW I am still perfectly able to connect to 10080 with FF and Safari as of yesterday.

**Comment by lmpater** [comment]
Hi,

Yeah… The problem seems to be exclusive to Chrome. I solved it (before filing the bug ;-) by simply changing the published port when launching the container.

The reason I created the bug was to share the information. I was concerned that new users may experience this issue and simply give up on Gogs because the example/tutorial suggests a port that doesn’t work for the most common browser. That would be a shame because Gogs rocks - thank you so much btw!

So take the bug as a “documentation bug”, not a software one ;-)

Cheers,
Marcelo
